volume/devicemapper/devmapper/deviceset.go

```diff
@@ -1127,7 +1129,9 @@ func (devices *DeviceSet) checkGrowBaseDeviceFS(info *devInfo) error {
 	}
 
 	if devices.baseFsSize < devices.getBaseDeviceSize() {
-		return fmt.Errorf("devmapper: Base device size cannot be smaller than %s", units.HumanSize(float64(devices.getBaseDeviceSize())))
+		return fmt.Errorf(
+			"devmapper: Base device size cannot be smaller than %s",
+			units.CustomSize("%.4g%s", float64(devices.getBaseDeviceSize()), 1024.0, sizeAbbrs))
```

Comment:
Can you confirm that when Docker parses this in the config file, it treats it as gibibytes rather than gigabytes?